<launch>
	
	<arg name="data_folder" default="data" />
	<arg name="list" default="cfg/test_1.list" />

  <node pkg="orp" type="load_item_params.py" name="load_item_params" args="$(find orp)/data/fall_2015/ycb.sqlite" output="screen" respawn="false" />

	<include file="$(find orp)/launch/orp.launch">
 		<arg name="realsense"          value="true"/>
 		<arg name="list"             value="$(arg list)"/>
 		<arg name="autostart"        value="true" />
		<arg name="rviz"             value="false" />
	</include>

  <node
    name    = "cylinder_classifier"
    pkg     = "orp"
    type    = "cylinder_classifier"
    args    = "true"

    respawn = "true"
    output  = "screen"
  />

  <include file="$(find orp)/launch/det_manip_settings.launch">
  </include>

</launch>
